### ISSUE TITLE
Initial conditions should have instructions & be selectable for all variables 

### ISSUE THREAD

**Issue by alexkaiser** [SimVascular/svZeroDSolver#141]
### Problem

The documentation does not make it clear how to set initial conditions. This is especially unclear for internal state variables such as the Windkessel pressure in an RCR bc or the chamber state variables. 

### Solution

The documentation should show examples of setting initial conditions for state variables. 

### Additional context

It is possible that some of these variables are initialized in some hard coded manner, to zero, estimated steady state only, etc. If this is the case, then code to set initial values for all possible variables should be added. 

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct and Contributing Guidelines

**Comment by alexkaiser** [comment]
@ktbolt Is there a consistent manner and naming scheme that we can add to the documentation accordingly?

**Comment by ktbolt** [comment]
@alexkaiser Are you referring to the web documentation ? That was just moved over from GitHub pages and is not well-organized or consistent.

**Comment by alexkaiser** [comment]
@ktbolt yes, it took me a really long time to figure out how to set initial conditions, which was important for my use case. This did not really seem to be documented on the web but would be a helpful and straightforward addition.

**Comment by ktbolt** [comment]
@alexkaiser OK, open an Issue in https://github.com/SimVascular/simvascular.github.io. If you have not added web documentation before then just add the text to the Issue and I can add it.
